searx/static/themes/simple/src/less/style.less

```diff
@@ -704,7 +704,7 @@ article.result-images[data-vim-selected]::before {
 
   #linkto_preferences {
     display: none;
-    postion: fixed !important;
+    postion: fixed !important; // stylelint-disable property-no-unknown
```

Comment:
There is a type `postion` --> `position` !

Reply:
LOL :-) .. here you can see that I am a dyslexic .. I rebased and dropped this commit 50495757, the fix is now in 5b7e7aa.

Reply:
Actually I compiled the theme, start the devtools, saw the property was not recognized, and then I realized the problem was not the value but the property name...